sdk/core/core-amqp/test/requestResponse.spec.ts

```diff
@@ -276,7 +276,7 @@ describe("RequestResponseLink", function() {
 
     const message = await retry<Message>(config);
     assert.equal(count, 2, "It should retry twice");
-    assert.equal(message == undefined, false, "It should return a valid message");
+    assert.equal(message === undefined, false, "It should return a valid message");
```

Comment:
I'm fixing this now.

Reply:
Actually, do we need to fix this? This assertion seems reasonable.

Reply:
A better way to do this (at least in chai) is assert.exists: https://www.chaijs.com/api/assert/#method_exists

Reply:
If we are making a change, then I prefer what @xirzec is recommending.

Reply:
Donezo!